lib/Doctrine/Common/Annotations/TokenParser.php

```diff
@@ -114,13 +116,18 @@ public function parseUseStatement()
                 $explicitAlias = true;
                 $alias = '';
             } else if ($token === ',') {
-                $statements[strtolower($alias)] = $class;
+                $statements[strtolower($alias)] = $groupRoot . $class;
                 $class = '';
                 $alias = '';
                 $explicitAlias = false;
             } else if ($token === ';') {
-                $statements[strtolower($alias)] = $class;
+                $statements[strtolower($alias)] = $groupRoot . $class;
                 break;
+            } else if ($token === '{' ) {
+                $groupRoot = $class;
+                $class = '';
+            } else if ($token === '}' ) {
+                continue;
```

Comment:
If PHP's grammar is modified to accept: `use Foo\{Bar, Woo}, Bar\{Fuzz, Lala};` then replace `continue;` with `$groupRoot = $class = $alias = '';`